lib/Frontend/ModuleInterfaceSupport.cpp

```diff
@@ -286,6 +286,14 @@ static void printImports(raw_ostream &out,
     allImportFilter |= ModuleDecl::ImportFilterKind::SPIOnly;
   }
 
+  llvm::SmallSet<ImportedModule, 8, ImportedModule::Order> packageOnlyImportSet;
+  if (!Opts.printPublicInterface() &&  Opts.printPackageInterface()) {
```

Comment:
Could we expect that if `Opts.printPackageInterface()` then `Opts.printPublicInterface()` is automatically false?

Bonus nitpick, there's a double space.